pages-includes/docs/tutorials/native/basic-kotlin-native-app-codeblocks-code.md

```diff
@@ -4,11 +4,11 @@
 
 ```
 plugins {
-    id 'org.jetbrains.kotlin.multiplatform' version '1.3.21'
+    id 'org.jetbrains.kotlin.multiplatform' version '1.3.72'
```

Comment:
Let's use the placeholder for the latest stable version `{{ site.data.releases.latest.version }`:
```suggestion
    id 'org.jetbrains.kotlin.multiplatform' version '{{ site.data.releases.latest.version }'
```